src/ast.ts

```diff
@@ -85,3 +91,7 @@ const termRegex = /[\w-]+/gm;
 export function getNumTerms(str: string): number {
   return str.match(termRegex)?.length ?? 0;
 }
+
+export function getDocumentNode(children = []): DocumentNodeProto {
```

Comment:
As written, I believe every default call gets passed the same array instance, meaning if any of them are mutated...all of them are. But also, consider dropping this whole function as suggested above.

Reply:
Nope, I'm wrong, that's c++

Reply:
I'm fine with moving this whole thing to the test helper anyway